packages/gamut/src/Markdown/__tests__/Markdown.test.tsx

```diff
@@ -78,47 +81,55 @@ describe('<Markdown />', () => {
 | col 2 is |    centered   |   $12 |
 | col 3 is | right-aligned |    $1 |
     `;
-    const { wrapper } = renderWrapper({
+    renderView({
       skipDefaultOverrides: { table: true },
       text: table,
     });
-    expect(wrapper.find('table').length).toEqual(1);
-    expect(wrapper.find('div.tableWrapper table').length).toEqual(0);
+    expect(document.querySelectorAll('table').length).toEqual(1);
+    expect(document.querySelectorAll('div.tableWrapper table').length).toEqual(
+      0
+    );
   });
 
   it('Wraps youtube iframes in a flexible container', () => {
-    const { wrapper } = renderWrapper({ text: youtubeMarkdown });
-    expect(wrapper.find('[data-testid="yt-iframe"]').length).toEqual(1);
+    renderView({ text: youtubeMarkdown });
+    expect(
+      document.querySelectorAll('[data-testid="yt-iframe"]').length
+    ).toEqual(1);
```

Comment:
this can be simplified to
```suggestion
view.getByTestId('yt-iframe')
```